src/layouts/general/layout.module.css

```
@@ -557,11 +557,20 @@
   }
   .nav__handbooks__container {
     overflow-x: hidden;
+    padding-right: 1rem;
     height: 100%;
   }
   .nav__handbooks__container::-webkit-scrollbar {
-    display: none;
+    background-color: var(--color-secondary1__shade2);
+    border-radius: 10px;
+    width: 10px;
```

Kan være lurt å holde seg til `rem` her for å skalere automatisk basert på font-størrelse på html

```suggestion
    border-radius: 0.625rem;
    width: 0.625rem;
```